src/public/parts/shells/code/public/index.html

```
@@ -0,0 +1,74 @@
+	<script src="https://cdn.jsdelivr.net/npm/@tailwindcss/browser@4" crossorigin="anonymous"></script>
+	<script type="module" src="/base.mjs"></script>
+	<link rel="stylesheet" href="./index.css" />
+</head>
+<body>
+	<div class="code-layout flex h-dvh w-full overflow-hidden bg-base-100 text-base-content">
+		<main class="code-main flex flex-col flex-1 min-w-0">
+			<h1 class="sr-only" data-i18n="code.title"></h1>
+			<header class="code-topbar flex items-center gap-2 px-2 py-0.5 border-b border-base-content/10">
+				<div class="code-topbar-left flex items-center gap-1.5 shrink-0 min-w-0 flex-1">
+					<div class="dropdown" id="home-wrap">
+						<button class="btn btn-ghost btn-square btn-sm" id="home-toggle" role="button" tabindex="0" data-i18n="code.home" aria-haspopup="true">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)